### ISSUE TITLE
Simple feature idea: URL generation

### ISSUE THREAD

**Issue by copumpkin** [AdRoll/hologram#37]
`hologram url` could write a federated access URL for your current hologram user to stdout. On a Mac, you could then run `hologram url | xargs open` and it would pop open a browser pointing at the AWS console.

The logic for this could live in pure client code and wouldn't need any protocol modifications. I'm pretty sure I could implement it in a handful of lines if you were interested.

My main concern about this is that the logic doesn't really feel specific to hologram. We'd just be running a standard STS operation against the current instance credentials. Putting it under hologram would thus be just for the sake of a consistent Amazon "user experience", rather than because it needs to be here. 

Thoughts?

**Comment by frangarciam** [comment]
I have a similar concern, as in we have to be careful not to bloat hologram with tons of options/features, but that does sound like it might be useful for some people without affecting the core functionality, so I'd be happy to review any PR you might want to send our way.

**Comment by walterking** [comment]
from boto import sts

import requests

import simplejson as json

import urllib

conn = sts.STSConnection()

role = conn.assume_role('<role arn>,'<profile name>')

session = {
"sessionId":role.credentials.access_key,"sessionKey":role.credentials.secret_key,"sessionToken":role.credentials.session_token}

params = { "Action":"getSigninToken","Session":json.dumps(session)}


r=requests.get("https://signin.aws.amazon.com/federation", params=params)

token=json.loads(r.text)['SigninToken']

params2 = { "Action":"login", "Issuer":"adroll.com","Destination":"
https://console.aws.amazon.com/console/home","SigninToken":token}


print "https://signin.aws.amazon.com/federation?{}
".format(urllib.urlencode(params2))


neat.  i just verified this works with hologram.

**Comment by copumpkin** [comment]
Yeah, I use something similar except with the current credentials instead
of another STS call. Just figured it would be smooth to add to the CLI :)

**Comment by ryansydnor** [comment]
I took a slightly different approach to the same problem and made a Chrome extension that leverages instance metadata to open the console. Closed this [issue](https://github.com/AdRoll/hologram/issues/67) by building [Holochrome](https://github.com/ryansydnor/holochrome).
